nomad/dns.nomad

```diff
@@ -0,0 +1,59 @@
+job "dns" {
+  datacenters = ["dc1"]
+  type        = "system"
+
+  group "dnsmasq" {
+    network {
+      mode = "bridge"
+      port "dns" {
+        static = 53
+        to     = 53
+      }
+    }
+
+
+    task "dnsmasq" {
+      driver = "docker"
+
+      config {
+        #This is an alpine-based dnsmasq container
+        image = "4km3/dnsmasq:2.85-r2"
```

Comment:
Long-term we'll want to switch this to either an official dnsmasq container or one that we build